ffmpeg-plugin/mcm_common.c

```diff
@@ -90,8 +90,28 @@ int mcm_parse_conn_param(AVFormatContext* avctx, MeshConnection *conn,
         cfg.interface_id = interface_id;
 
         err = mesh_apply_connection_config_memif(conn, &cfg);
-        if (err)
+        if (err) {
+            av_log(avctx, AV_LOG_ERROR, "Failed to apply memif configuration: %s (%d)\n",
+                mesh_err2str(err), err);
```

Comment:
No need to add this log print since it's properly printed at the caller level
```cpp
    err = mcm_parse_conn_param(avctx, s->conn, kind, s->ip_addr, s->port,
                               s->protocol_type, s->payload_type,
                               s->socket_name, s->interface_id);
    if (err) {
        av_log(avctx, AV_LOG_ERROR, "Configuration parsing failed: %s (%d)\n",
               mesh_err2str(err), err);
        err = AVERROR(EINVAL);
        goto exit_delete_conn;
    }
```